packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -14,134 +14,94 @@ void main() {
   testWidgets('BottomNavigationBar callback test', (WidgetTester tester) async {
     int mutatedIndex;
 
-    await tester.pumpWidget(
-      MaterialApp(
+    await tester.pumpWidget(MaterialApp(
         home: Scaffold(
-          bottomNavigationBar: BottomNavigationBar(
-            items: const <BottomNavigationBarItem>[
-              BottomNavigationBarItem(
-                icon: Icon(Icons.ac_unit),
-                title: Text('AC')
-              ),
-              BottomNavigationBarItem(
-                icon: Icon(Icons.access_alarm),
-                title: Text('Alarm')
-              )
-            ],
-            onTap: (int index) {
-              mutatedIndex = index;
-            }
+            bottomNavigationBar: BottomNavigationBar(
+                items: const <BottomNavigationBarItem>[
+          BottomNavigationBarItem(
+            icon: Icon(Icons.ac_unit),
+            title: Text('AC'),
+          ),
+          BottomNavigationBarItem(
+            icon: Icon(Icons.access_alarm),
+            title: Text('Alarm'),
           )
-        )
-      )
-    );
+        ],
+                onTap: (int index) {
+                  mutatedIndex = index;
+                }))));
 
     await tester.tap(find.text('Alarm'));
 
     expect(mutatedIndex, 1);
   });
 
   testWidgets('BottomNavigationBar content test', (WidgetTester tester) async {
-    await tester.pumpWidget(
-      MaterialApp(
+    await tester.pumpWidget(MaterialApp(
         home: Scaffold(
-          bottomNavigationBar: BottomNavigationBar(
-            items: const <BottomNavigationBarItem>[
-              BottomNavigationBarItem(
-                icon: Icon(Icons.ac_unit),
-                title: Text('AC')
-              ),
-              BottomNavigationBarItem(
-                icon: Icon(Icons.access_alarm),
-                title: Text('Alarm')
-              )
-            ]
-          )
-        )
-      )
-    );
+            bottomNavigationBar:
+                BottomNavigationBar(items: const <BottomNavigationBarItem>[
```

Comment:
Unfortunately a bunch of these turned out a bit wonky. Either revert the auto-format and just keep the change to the addition of the new test or (expanding the scope a bit unfortunately) add trailing commas so the formatter keeps the general shape cascading.